doc/source/data/transforming-data.rst

```diff
@@ -124,10 +124,10 @@ To transform batches with actors, complete these steps:
 
 1. Implement a class. Perform setup in ``__init__`` and transform data in ``__call__``.
 
-2. Create an :class:`~ray.data.ActorPoolStrategy` and configure the number of concurrent
-   workers. Each worker transforms a partition of data.
+2. Configure ``concurrency`` with the number of concurrent workers. Each worker
+transforms a partition of data.
 
-3. Call :meth:`~ray.data.Dataset.map_batches` and pass your ``ActorPoolStrategy`` to ``compute``.
+3. Call :meth:`~ray.data.Dataset.map_batches`.
```

Comment:
I think we should also restructure this doc page. In particular, I'd like to have separate sections for "Choosing between tasks and actors" and "controlling concurrency", rather than putting them under "Transforming batches". Because they are not specific to "map_batches". Also, maybe emphasize terms like "stateful/stateless", and downplay terms like "tasks/actors".

Reply:
Yes make sense to me. I restructure the doc to create a separate section for "Transforming with Python Class". Emphasize on function and class. Add explain that Ray Data uses task to execute function, and uses actor to execute class.

I don't create a section for "controlling concurrency", because we are not recommending users to use auto-scaling actor pool. We can create it later when we have full story of task and actor pool concurrency control.